arch/arm/src/common/arm_createstack.c

```diff
@@ -220,7 +220,8 @@ int up_create_stack(FAR struct tcb_s *tcb, size_t stack_size, uint8_t ttype)
       /* Save the adjusted stack values in the struct tcb_s */
 
       tcb->adj_stack_ptr  = (uint32_t *)top_of_stack;
-      tcb->adj_stack_size = size_of_stack;
+      tcb->adj_stack_size = size_of_stack -
```

Comment:
@johannes-nivus could you apply the same change to other arch?

Reply:
I could, but I cannot test.
Perhaps we should wait for some more comments concerning the change in general...

Reply:
The same change would be needed in arm_usestack.c.

adj_stack_size is also set in arm_vfork.c, I would need to review to understand what, if anything, is needed.

Reply:
Yes you're right. I missed arm_usestack.c. But I think arm_vfork.c is save. (I don't see it beeing set there).

Reply:
> But I think arm_vfork.c is save. (I don't see it beeing set there).

I was surprised to see it there too.  Turns out, my grep was just bad and caught this link in arm_vfork.c:

   stacksize = parent->adj_stack_size + CONFIG_STACK_ALIGNMENT - 1;

It has bother adj_stack_size and " = ".  Sorry I was not careful.  My mistake.

Reply:
https://github.com/apache/incubator-nuttx/blob/b256b2055f7e519104052499e6855013cb06cc5f/arch/arm/src/common/arm_createstack.c#L237-L238
Then, this needs to be adjusted too?  No need to take sizeof(tls_info_s) off anymore.